examples/cassandra/README.md

```diff
@@ -300,47 +226,106 @@ spec:
+We also tell Kubernetes that the container exposes
+both the `CQL` and `Thrift` API ports.  Finally, we tell the cluster
+manager that we need 0.1 cpu (0.1 core).
+
+Create the Replication Controller:
 
 ```console
 $ kubectl create -f examples/cassandra/cassandra-controller.yaml
 ```
 
-Now this is actually not that interesting, since we haven't actually done
-anything new.  Now it will get interesting.
-
-Let's scale our cluster to 2:
+You can list the new controller:
 
 ```console
-$ kubectl scale rc cassandra --replicas=2
+$ kubectl get rc -o wide
+NAME        DESIRED   CURRENT   AGE       CONTAINER(S)   IMAGE(S)                             SELECTOR
+cassandra   2         2         11s       cassandra      gcr.io/google-samples/cassandra:v8   app=cassandra
 ```
 
-Now if you list the pods in your cluster, and filter to the label `app=cassandra`, you should see two cassandra pods:
+Now if you list the pods in your cluster, and filter to the label
+`app=cassandra`, you should see two Cassandra pods. (The `wide` argument lets
+you see which Kubernetes nodes the pods were scheduled onto.)
 
 ```console
-$ kubectl get pods -l="app=cassandra"
-NAME              READY     STATUS    RESTARTS   AGE
-cassandra         1/1       Running   0          3m
-cassandra-af6h5   1/1       Running   0          28s
+$ kubectl get pods -l="app=cassandra" -o wide
+NAME              READY     STATUS    RESTARTS   AGE       NODE
+cassandra-21qyy   1/1       Running   0          1m        kubernetes-minion-b286
+cassandra-q6sz7   1/1       Running   0          1m        kubernetes-minion-9ye5
 ```
 
-Notice that one of the pods has the human-readable name `cassandra` that you
-specified in your config before, and one has a random string, since it was named
-by the replication controller.
+Because these pods have the label `app=cassandra`, they map to the service we
+defined in Step 1.
+
+You can also query the service endpoints to check if the pod has been correctly
+selected, like this (substituting `cassandra-xxxxx` with the actual name of one
+of your pods):
+
+```console
+$ kubectl get endpoints cassandra-xxxx -o yaml
+apiVersion: v1
+kind: Endpoints
+metadata:
+  creationTimestamp: 2015-06-21T22:34:12Z
+  labels:
+    app: cassandra
+  name: cassandra
+  namespace: default
+  resourceVersion: "944373"
+  selfLink: /api/v1/namespaces/default/endpoints/cassandra
+  uid: a3d6c25f-1865-11e5-a34e-42010af01bcc
+subsets:
+- addresses:
+  - ip: 10.244.3.15
+    targetRef:
+      kind: Pod
+      name: cassandra
+      namespace: default
+      resourceVersion: "944372"
+      uid: 9ef9895d-1865-11e5-a34e-42010af01bcc
+  ports:
+  - port: 9042
+    protocol: TCP
+```
 
-To prove that this all works, you can use the `nodetool` command to examine the
-status of the cluster.  To do this, use the `kubectl exec` command to run
-`nodetool` in one of your Cassandra pods.
+To show that the `SeedProvider` logic is working as intended, you can use the
```

Comment:
Let's just leave this for now, and address it in a future iteration (meaning, I'm not sure, but as per our email thread on this, there are indications that the current example may not be using best practices with the SeedProvider.)